locale/ar/locale.po

```diff
@@ -28,6 +28,7 @@ msgstr "إذا غيرت مراجع المقالة بعد تسجيل معرفات
 msgid "plugins.generic.crossrefReferenceLinking.description.requirements"
 msgstr "هذه الإضافة تتطلب تتطلب أن تكون المقالة قد حصلت على معرف DOI لـ Crossref وأن هذا المعرف لا بد أن يكون مودعاً مع Crossref من داخل نظام المجلات المفتوحة. مراجع المقالة يمكن الحصول عليها من باب البيانات الوصفية ضمن مخطط سير العمل لطلب التقديم."
 
+#, fuzzy
```

Comment:
Given the small number of locale files, perhaps @asmecher has to add this one to Weblate.

Reply:
I can add it after this gets merged (so the files are in the new directory structure); @bozana, can you ping me when this is merged?